src/index.ts

```diff
@@ -475,6 +495,7 @@ class SenseiClient {
     return {
       transactions: transactions.map((transaction: any) => {
         return {
+          transaction: transaction.transaction,
```

Comment:
don't you also need to perform the mapping to camelCase here?

Reply:
On `transaction`? Where?

Reply:
you mapped all of the transaction's nested fields to camelCase (e.g. inside of TransactionInput and TransactionOutput and even lockTime on tx itself) but I think they all come back as snake_case from the sensei api, right?

so you'll need to do something mapping the transaction to a camelCase'd object?

Reply:
Oh yeah, obviously, my bad. Will push this today.